deployment/src/main/resources/templates/libraries/microprofile/pojo.qute

```diff
@@ -9,6 +9,8 @@ import jakarta.xml.bind.annotation.XmlEnumValue;
 {#else}
 import java.lang.reflect.Type;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
```

Comment:
Rather than adding new imports, we've been referencing the FQCN to avoid conflicts with existing classes that may have the same name.

Reply:
@hbelmiro All the xml classes are imported and to be honest I do not think thats a good policy, since it makes the generated class pretty difficult to read. If there is a conflict (very unlikely anyway), we will see it during ci cycle.

Reply:
I know there are several imports, but we've been avoiding adding new ones. There's an open [issue to remove the existing imports](https://github.com/quarkiverse/quarkus-openapi-generator/issues/103).
The conflict that may happen is with a class from the OpenAPI document. Imagine it has a custom type called `Include` defined in the OpenAPI document. The CI wouldn't get these cases.
It's very unlikely, but [it happened once](https://github.com/quarkiverse/quarkus-openapi-generator/pull/102).